shor/framework/docstring.py

```diff
@@ -0,0 +1,54 @@
+from abc import ABCMeta
+from enum import Enum
+from inspect import getdoc
+from textwrap import indent
+
+
+class _Hints(Enum):
```

Comment:
Goal for this class? I don't see an issue with it, just curious.

Reply:
I used this once, it might not be needed, but it was for reusing docstring comments.

I had one for login that I used in a couple of places in IBMQ, so decided it would be nice to share the same text in multiple places

Reply:
I used an underscore to signal to outside uses of the API that this is not a class we intend for them to use